pkg/auth/audit/audit.go

```diff
@@ -78,6 +96,17 @@ func getUserInfo(req *http.Request) *User {
 	}
 }
 
+func getUserNameForBasicLogin(body []byte) string {
```

Comment:
This appears to have gotten pulled over in the backport. In 2.6, this appears to be used in [newAuditLog](https://github.com/KevinJoiner/rancher/blob/43660c90aaf2e8b14447eea7d10fab818ab46829/pkg/auth/audit/audit.go#L140). However, 2.5 does not appear to [do to the same](https://github.com/rancher/rancher/blob/96b05b53a48d7c4cc4a9ae5eb4c7353547757af7/pkg/auth/audit/audit.go#L129).

I would recommend one of two paths:
- We port over this new function and the place which uses the function
- We do not port over this function or it's usage at all

LMK what you think.

Reply:
I didn't mean to add this function so option b 'We do not port over this function or it's usage at all'